pkgs/build-support/setup-hooks/auto-patchelf.sh

```diff
@@ -167,9 +153,14 @@ autoPatchelf() {
     # findDependency outside of this, the dependency cache needs to be rebuilt
     # from scratch, so keep this in mind if you want to run findDependency
     # outside of this function.
-    findElfs "$prefix" | while read -r elffile; do
-        autoPatchelfFile "$elffile"
-    done
+    while IFS= read -r -d $'\0' file; do
```

Comment:
Why the `IFS=`?

Reply:
taken from here: http://mywiki.wooledge.org/BashFAQ/020

Reply:
The page says it is required.

Reply:
Ah, you're right, otherwise it's trimmed by `IFS`.